features/app/native/navigationHandlers.js

```diff
@@ -0,0 +1,49 @@
+import Config from '../../../config';
+
+import {
+    destroy,
+    init
+} from '../../base/connection';
+import { Conference } from '../../conference';
+import { WelcomePage } from '../../welcome';
+
+import { APP_SCREEN } from '../constants';
+
+const navigationHandlers = {};
+
+/**
+ * Handler for APP_SCREEN.WELCOME screen route. Saves root route in navigation.
+ * Destroy connection, conference and local tracks.
+ *
+ * @param {Store} store - Redux store.
+ * @param {Object} action - Action object.
+ * @param {Navigator} action.navigator - Navigator instance.
+ * @returns {void}
+ */
+navigationHandlers[APP_SCREEN.WELCOME] = (store, action) => {
+    action.navigator.push({
+        title: 'Jitsi Meet',
+        component: WelcomePage
```

Comment:
Please comply with the coding style which requires sorting by alphabetical order!